modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/odata/filter/expression/SOSQLExpressionVisitorImpl.java

```diff
@@ -140,6 +146,12 @@ public Object visitLiteralExpression(LiteralExpression literalExpression)
 				StringPool.APOSTROPHE);
 		}
 
+		if (Objects.equals(
```

Comment:
Can you sort it? Put this if checking before the string one. Thx

Reply:
done!